packages/tocco-ui/src/Menu/StyledComponents.js

```diff
@@ -13,6 +13,10 @@ export const StyledIconButtonWrapper = styled(Button)`
   padding-right: .9rem;
 `
 
+export const StyledPopperRef = styled.div`
```

Comment:
Why "Ref"?  Do you apply position relative because of the react ref prop?
I would rename it